### PR TITLE
17 experience conclusion

### PR DESCRIPTION
**Pull request by tom08**

### CODE REVIEW

File: exdb/templates/exdb/conclusion.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                </p>
+                <p>
+                    <span>{% trans "Guest Office" %}:</span>
+                    {{ experience.guest_office }}
+                </p>
+            {% endif %}
+        </div>
+        <hr/>
+        <div>
+            {% for field in form %}
```
Comment (enki211) [inline exdb/templates/exdb/conclusion.html]: Indentation

**Comment by enki211** [comment]
We should probably show the comments on the experience view page

**Comment by shaneallgeier** [comment]
In the next sprint, the plan is to merge the edit/view Experience pages all into one template. In which case, the comments would be a part of that.

File: exdb/templates/exdb/conclusion.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            <p>{{ field.errors }}</p>
+            <div>
```
Comment (shaneallgeier) [inline exdb/templates/exdb/conclusion.html]: This `<div>` is never closed.

File: exdb/templates/exdb/ra_home.html [hunk 3]
```diff
@@ -15,21 +15,21 @@
...
-                        <div class="description">
-                            <fieldset>
-                                <legend>{% trans 'Description' %}</legend>
-                                <div class="description-text">{{ ex.description }}</div>
-                            </fieldset>
-                        </div>
-                        <div class="experience-footer">
-                            {% trans 'Created on' %}: {{ ex.created_datetime }}
-                        </div>
+                <div class="experience"><a href={% if ex.needs_evaluation %}{% url 'conclusion' pk=ex.pk %}{% else %}{% url 'view_experience' pk=ex.pk %}{% endif %}>
```
Comment (shaneallgeier) [inline exdb/templates/exdb/ra_home.html]: Indentation

File: exdb/templates/exdb/conclusion.html (hunk 1 above)
Comment (tom08) [inline exdb/templates/exdb/conclusion.html]: This issue has now been addressed.

**Comment by enki211** [comment]
Looks Good

**Comment by tom08** [comment]
I approve of what I did now write.

**Comment by shaneallgeier** [comment]
I approve.

**Pull request merged**